index.html

```diff
@@ -0,0 +1,101 @@
+
+        <link href="styles.css" rel="stylesheet">
+
+        <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+        <!--[if lt IE 9]>
+          <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+          <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+        <![endif]-->
+    </head>
+
+    <body>
+
+        <nav class="navbar navbar-inverse navbar-fixed-top">
+            <div class="container">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="#">examples.sdi4apps.eu</a>
+                </div>
+                <div id="navbar" class="navbar-collapse collapse">
+                    <!--form class="navbar-form navbar-right">
+                        <div class="form-group">
+                            <input type="text" placeholder="Email" class="form-control">
+                        </div>
+                        <div class="form-group">
+                            <input type="password" placeholder="Password" class="form-control">
+                        </div>
+                        <button type="submit" class="btn btn-success">Sign in</button>
+                    </form-->
+                </div><!--/.navbar-collapse -->
+            </div>
+        </nav>
+
+        <!-- Main jumbotron for a primary marketing message or call to action -->
+        <div class="jumbotron">
+            <div class="container">
+                <h1>Introducing s4a.js</h1>
+                <p>s4a.js is a client-side JavaScript library for rapid application development based on OpenLayers 3 and D3.js. It is developed by the <a href="http://www.sdi4apps.eu">SDI4Apps consortium</a>, co-funded by the European Commission, and is currently in the stage of implementation. The first public release is planned for April 2016.</p>
+                <p><a class="btn btn-primary btn-lg" href="https://github.com/SDI4Apps/s4a.js" role="button">s4a.js on GitHub &raquo;</a></p>
+            </div>
+        </div>
+
+        <div class="container">
+            <!-- Example row of columns -->
+            <div class="row">
+                <div class="col-md-4">
+                    <h2>License</h2>
+                    <p>the library is released by the SDI4Apps consortium under an Apache 2.0 license</p>
+                    <p><a class="btn btn-default" href="LICENSE" role="button">View license &raquo;</a></p>
```

Comment:
I do not think it a good idea to duplicate LICENSE information into another file, setting the configuration for IIS to serve extension-less files as text/html is easy:

```
<staticContent>
   <mimeMap fileExtension="." mimeType="text/html" />
</staticContent>
```